com.unity.netcode.gameobjects/Runtime/AssemblyInfo.cs

```diff
@@ -10,3 +10,4 @@
 [assembly: InternalsVisibleTo("TestProject.RuntimeTests")]
 [assembly: InternalsVisibleTo("Unity.Netcode.RuntimeTests")]
 [assembly: InternalsVisibleTo("Unity.Netcode.TestHelpers.Runtime")]
+[assembly: InternalsVisibleTo("Unity.Netcode.Adapter.UTP")]
```

Comment:
this needed to be put back to prevent projects with UTP Adapter package installed from breaking.